### ISSUE TITLE
ZABBIX NOT SUPPORTED KEY

### ISSUE THREAD

**Issue by MadameGit** [Relkci/Zabbix_Bandwidth-vnstat#1]
Hello,

I am trying to monitor monthly bandwidth client using the zabbix agent, vnstat and UserParameters.
I installed vnstat, configure Zabbix Agent (user parameters), import template and assign host.
But, I have a problem.
When I run the "vnstat -u -i ens160" and "vnstat -m" on the client, I get a correct value.
However, from the zabbix server returns "NOT SUPPORTED KEY" on this key : "bandwidth.vnstat.discovery.int.count" and Dashboard does not return anything.

Did someone face something similar?

![graph](https://user-images.githubusercontent.com/91667406/135440217-e595f61c-f761-4ef9-9098-2a1bee4fb28b.PNG)
![error](https://user-images.githubusercontent.com/91667406/135440269-fd398ec7-0838-4e2b-b19a-9d65d09ec976.PNG)

**Comment by Relkci** [comment]
Can you confirm you are using an Active Agent and not Passive Agent?  If you are using Passive, you will need to change the parent discovery item to be the standard Agent instead of Active Agent.

You can also change the “retain history” parameter for the discovery parent item, this will confirm if Zabbix was able to gather any data from vnstat.  Once the parent item has data, the discovery template should build the rest of the items.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: MadameGit ***@***.***>
Sent: Thursday, September 30, 2021 4:38:30 AM
To: Relkci/Zabbix_Bandwidth-vnstat ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Relkci/Zabbix_Bandwidth-vnstat] ZABBIX NOT SUPPORTED KEY (#1)


Hello,

I am trying to monitor monthly bandwidth client using the zabbix agent, vnstat and UserParameters.
I installed vnstat, configure Zabbix Agent (user parameters), import template and assign host.
But, I have a problem.
When I run the "vnstat -u -i ens160" and "vnstat -m" on the client, I get a correct value.
However, from the zabbix server returns "NOT SUPPORTED KEY" on this key : "bandwidth.vnstat.discovery.int.count" and Dashboard does not return anything.

Did someone face something similar?

[graph]<https://user-images.githubusercontent.com/91667406/135440217-e595f61c-f761-4ef9-9098-2a1bee4fb28b.PNG>
[error]<https://user-images.githubusercontent.com/91667406/135440269-fd398ec7-0838-4e2b-b19a-9d65d09ec976.PNG>

**Comment by MadameGit** [comment]
Even with Active Agent, the same problem remains : empty graph and an error occurs in some items.
![item-error](https://user-images.githubusercontent.com/91667406/135579669-0359c106-0c52-4e7c-bfb8-12f2da68cd2a.PNG)

**Comment by Relkci** [comment]
Can you see if recent commit fixes the issue?

**Comment by MadameGit** [comment]
Yes the graph problem has been solved but there is still an error in some items.

**Comment by MadameGit** [comment]
I just have a problem with "Monthly ens160 Bandiwdth RX Graph" ==> No Data

![Rx Monthly](https://user-images.githubusercontent.com/91667406/136169375-35d1cb87-da48-4511-8b59-c4b3869033f9.PNG)
![Monthly Rx](https://user-images.githubusercontent.com/91667406/136169427-a6dcb3ab-29a5-456c-9760-7da12fc31bee.PNG)
